stylus/components/nav.styl

```diff
@@ -126,3 +130,45 @@ $nav-link
     @media not all and (pointer: fine)
         &:hover
             color var(--slateGrey)
+
+
+$nav-item-secondary
+    height auto
+    +medium-screen()
+        &
+            display none
+
+    &
```

Comment:
We can simplify this blog by removing the ampersands and moving the margin-top declaration.

Reply:
Done